### PR TITLE
Enable verification of special JWT payload fields (closes #668).

### LINKED ISSUE #668: Enable verification of special JWT payload fields

### PR DESCRIPTION
**Pull request by yivo**

### CODE REVIEW

File: app/api/api_v2/auth/jwt_authenticator.rb [hunk 1]
```diff
@@ -37,6 +37,25 @@ def fetch_email(payload)
 
         email
       end
+
+      def token_verification_options
+        { verify_expiration: true,
+          verify_not_before: true,
+          iss:               ENV['JWT_ISSUER'].to_s.squish.split(','),
+          verify_iss:        ENV['JWT_ISSUER'].present?,
```
Comment (mod) [inline app/api/api_v2/auth/jwt_authenticator.rb]: only one issuer, not multiples

Comment (mod) [reply app/api/api_v2/auth/jwt_authenticator.rb]: it can be account.domain.com

**Review by mod — CHANGES_REQUESTED** [review]
looks good

File: app/api/api_v2/auth/jwt_authenticator.rb (hunk 1 above)
Comment (yivo) [reply app/api/api_v2/auth/jwt_authenticator.rb]: @mod the reason I make it an array is that JWT allow to check for multiple issuers

image

Will we keep array or refactor to single issuer?

**Pull request merged**